output/sqlite.go

```diff
@@ -0,0 +1,422 @@
+// +build cgo_sqlite
```

Comment:
I know at the moment we don't give the possibility to use another sqlite driver.
But... in theory the sqlite baker output shouldn't care about which particular sqlite driver is used under the hood so I tihnk it would be best that cgo does not appear here.

Generally for this kind of stuff, we'd have something along the lines of:


 - sqlite_cgo.go:

```go
// +build cgo_sqlite
package output

import (_ "github.com/mattn/go-sqlite3")
```

- sqlite_go.go:
```go
// +build !cgo_sqlite
package output

import (_ "/whatever/pure-go/package")
```

If we do this, we'll always have one, but only one, sqlite driver that is loaded, so we don't need to conditionally add `SQLiteDesc` to the `output.All`. 

Again, even if we don't have a pure-go package at the moment, we should still do this, but of course omit loading the package in `sqlite_go.go`. That means that if somebody tries to compile without providing `cgo_sqlite` it will just work, until they try to use the sqlite output, it will fail at startup saying it can't find an existing sqlite driver (which is exactly what we want)